token/cli/src/main.rs

```diff
@@ -2613,6 +2621,15 @@ fn app<'a, 'b>(
                             Rate authority defaults to the mint authority."
                         ),
                 )
+                .arg(
+                    Arg::with_name("metadata_address")
+                        .long("metadata-address")
+                        .value_name("METADATA_ADDRESS")
+                        .takes_value(true)
+                        .help(
+                            "Specify metadata address to use."
```

Comment:
nit: let's make this a bit clearer
```suggestion
                            "Specify address that stores token metadata."
```